mirrord/layer/src/lib.rs

```diff
@@ -85,8 +85,7 @@ static GUM: LazyLock<Gum> = LazyLock::new(|| unsafe { Gum::obtain() });
 
 pub(crate) static mut HOOK_SENDER: Option<Sender<HookMessage>> = None;
 
-pub(crate) static ENABLED_FILE_OPS: OnceLock<bool> = OnceLock::new();
-pub(crate) static ENABLED_FILE_RO_OPS: OnceLock<bool> = OnceLock::new();
+pub(crate) static FILE_MODE: OnceLock<FsConfig> = OnceLock::new();
```

Comment:
Better name, but considering that there is a "mode" for opening files, maybe we could have it be less ambiguous, like `FILE_OPS_MODE`?

Reply:
I like that it's the same name as the argument/env etc.. Leaving asi s